_maps/map_files/BoxStation/BoxStation.dmm

```diff
@@ -54107,12 +54145,40 @@
 /obj/machinery/door/firedoor,
 /turf/open/floor/plasteel,
 /area/quartermaster/miningdock)
-"eHI" = (
-/obj/machinery/atmospherics/pipe/manifold/supply/hidden{
-	dir = 8
+"eyS" = (
+/obj/machinery/atmospherics/components/unary/tank/carbon_dioxide{
+	desc = "What the fuck is this?"
 	},
-/turf/open/floor/plasteel/floorgrime,
-/area/maintenance/disposal/incinerator)
+/obj/item/bikehorn/airhorn{
+	pixel_x = 7;
+	pixel_y = 7
+	},
+/turf/open/floor/plating,
+/area/maintenance/port/fore)
+"eAp" = (
+/obj/machinery/door/airlock/engineering/glass{
+	name = "Engineering";
+	req_access_txt = "32";
+	security_level = 6
```

Comment:
just double checking, you do know 6 means electrified, right?

Reply:
Yea, it was intended before but I've taken a lot of the fun stuff out of the storage room now so I should bring it down to something more appropriate.